devops/build-push-featurebranch.sh

```diff
@@ -8,6 +8,8 @@ GIT_BRANCH=$(git branch | grep \* | cut -d ' ' -f2 | tr '[:upper:]' '[:lower:]')
 
 DOCKER_REPO=$REPOSITORY/$SERVER_NAME/$GIT_BRANCH
 SEM_VER=${SEM_VER=0.0.0}
+ARCH=amd64
```

Comment:
maybe can be passed as param with default (like SEM_VER)
or completely take from user env (so it will work by default for "arm" users)

Reply:
@gadotroee I'm not a user of this script and I don't really understand its purpose. Feel free to modify according to your development workflow.